.travis.yml

```diff
@@ -1,24 +1,51 @@
 # Copyright (c) 2015-2016, NVIDIA CORPORATION.  All rights reserved.
 
-sudo: required
+os: linux
 dist: trusty
-
+sudo: required
 language: python
 python: 2.7
 
 env:
+    global:
+        - CAFFE_ROOT=~/caffe
+        - TORCH_ROOT=~/torch
+        # Fixes for Torch and OpenBLAS
+        - OMP_NUM_THREADS=1
+        - OPENBLAS_MAIN_FREE=1
     matrix:
         - DIGITS_TEST_FRAMEWORK=caffe CAFFE_FORK=NVIDIA
         - DIGITS_TEST_FRAMEWORK=caffe CAFFE_FORK=BVLC
         - DIGITS_TEST_FRAMEWORK=torch
         - DIGITS_TEST_FRAMEWORK=none
-        - DEB_BUILD=true
+
+matrix:
```

Comment:
I don't really understand how this second definition of `matrix` works in conjunction with the first one. Should the one on line `16` not start with an `include` directive and be merged with this one on line `22`?